Intersect.Editor/Localization/Strings.cs

```diff
@@ -3749,6 +3753,8 @@ public partial struct ItemEditor
 
             public static LocalizedString SingleUseEvent = @"Destroy On Use?";
 
+            public static LocalizedString stackoptions = @"Stackable?";
```

Comment:
Since you are adding this it should be in PascalCase but the name would be better also as `Stackable`.